src/charm.py

```diff
@@ -63,10 +63,19 @@ def __init__(self, *args: typing.Any):
         self.framework.observe(
             self.on[constants.PEER].relation_joined, self._on_peer_relation_joined
         )
-        self.unit.open_port("tcp", 8080)  # ACL API
         self.unit.open_port("tcp", 53)  # Bind DNS
         self.unit.open_port("udp", 53)  # Bind DNS
 
+        # Try to check if the `charmed-bind-snap` resource is defined.
```

Comment:
Perhaps reword this to `Record if the ... resource is defined`. From reading `Try to check` I'm not sure what is meant by that. The code seems to set `self.snap_path` if the resource is defined

Reply:
reworded